data/reusables/desktop/open-repository-settings.md

```diff
@@ -0,0 +1,14 @@
+1. If you want to change how a fork behaves with the upstream repository, open the **Repository** menu, then click **Repository settings...**.
```

Comment:
Let's move the conceptual part...

>If you want to change how a fork behaves with the upstream repository,

Above the numbered steps. We should describe the purpose of the procedure before the procedure begins so folks can quickly tell if they're in the right place. That also allows us to use this reusable for any setting in this menu in the future.